src/appleseed/renderer/kernel/lighting/lighttypes.h

```diff
@@ -50,6 +45,12 @@ namespace renderer  { class Material; }
 namespace renderer
 {
 
+enum LightTypes
```

Comment:
Now that you're (appropriately) using this as a type, I would suggest renaming it to `LightType`.